docs/source/beam.mdx

```diff
@@ -44,3 +57,20 @@ datasets-cli run_beam datasets/$DATASET_NAME \
 When you run your pipeline, you can adjust the parameters to change the runner (Flink or Spark), output location (S3 bucket or HDFS), and the number of workers.
 
 </Tip>
+
+5. Example: running the wikipedia datasets pipeline on DataFlow
+
+```
+datasets-cli run_beam ${DATASET_NAME} \
+--name 20220301.fr \
+--language fr \
+--date 20230601 \
+--save_info \
+--cache_dir gs://$BUCKET/cache/datasets \
+--beam_pipeline_options=\
+"runner=DataflowRunner,project=${PROJECT},job_name=${DATASET_NAME}-gen,"\
+"staging_location=gs://${BUCKET}/binaries,temp_location=gs://${BUCKET}/temp,"\
+"region=${REGION},requirements_file=/tmp/beam_requirements.txt"
+```
+
+Maybe a bit counter-intuitive is that you have to specify the current config name (`20220301.fr`) and then override it with the `language` and `date` arguments. This allows the dataset script [wikipedia](https://huggingface.co/datasets/wikipedia/blob/main/wikipedia.py) to have a valid config. That script allows a specific date and language to be specified, but only on top of a valid configuration.
```

Comment:
Would it be possible to not ask for the config name when some extra config_kwargs are passed ?

Because right now it would use the `20220301.fr` config and modify it to match the extra config_kwargs, instead of creating a new config on-the-fly